components/centos/centos-mesos-marathon-singlenode-setup/Vagrantfile

```diff
@@ -0,0 +1,26 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+	config.vm.box = "centos/7"
```

Comment:
@dharmit K8s RPMs are installed in the ADB Vagrant box, but the services are not started by default.  So you should not face any issue starting Marathon services.